collector/fixtures/e2e-64k-page-output.txt

```diff
@@ -1664,6 +1664,51 @@ node_netstat_Udp_NoPorts 120
 # HELP node_netstat_Udp_OutDatagrams Statistic UdpOutDatagrams.
 # TYPE node_netstat_Udp_OutDatagrams untyped
 node_netstat_Udp_OutDatagrams 53028
+# HELP node_network_address_assign_type address_assign_type value of /sys/class/net/<iface>.
+# TYPE node_network_address_assign_type gauge
+node_network_address_assign_type{interface="eth0"} 3
+# HELP node_network_carrier carrier value of /sys/class/net/<iface>.
+# TYPE node_network_carrier gauge
+node_network_carrier{interface="eth0"} 1
+# HELP node_network_carrier_changes_total carrier_changes_total value of /sys/class/net/<iface>.
```

Comment:
`carrier_changes_total value of` -> ` carrier_changes value of` please fix it. And others in this PR